client/src/views/lander.js

```diff
@@ -0,0 +1,187 @@
+import Snabbdom from 'snabbdom-pragma'
+import layout from './layout'
+
+const staticRoot = process.env.STATIC_ROOT || ''
+
+const LandingPage = ({ t, ...S }) => layout(
+  <div className="landing-page">   
+  <script src="/js/infinite-scroll.js" async></script>   
```

Comment:
Should use `STATIC_ROOT` to refer to static file paths

Reply:
updated